packages/hooks/src/useHasMFAccountDeposited.ts

```diff
@@ -0,0 +1,27 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useHasMFAccountDeposited = () => {
```

Comment:
@farhan-nurzi-deriv Please help to make use of `TSDoc` and comment steps in the hook 🙏🏻 You can refer to [useWalletsList](https://github.com/binary-com/deriv-app/blob/7190dee6042ce51b9ad9d20a653d6e3f9fcdddcc/packages/hooks/src/useWalletsList.ts) hook as an example 🙇🏻